src/infra/docs/rustc-ci.md

```diff
@@ -63,25 +76,39 @@ Since the merge commit is based on the latest master and only one can be tested
 at the same time, when the results are green master is fast-forwarded to that
 merge commit.
 
+The `auto` branch and other branches used by bors live on a fork of the rust-lang/rust: [rust-lang-ci/rust]. This as originally done to some security limitations in GitHub Actions. These limitations have been addressed, but we've not yet done the work of removing the use of a fork.
+
 Unfortunately testing a single PR at the time, combined with our long CI (~3
-hours for a full run), means we can’t merge too many PRs in a single day, and a
+hours for a full run)[^1], means we can’t merge too many PRs in a single day, and a
 single failure greatly impacts our throughput for the day. The maximum number
 of PRs we can merge in a day is around 8.
 
+The large CI run times and requirement for a large builder pool is worth it because it: 
```

Comment:
Aren't these benefits of always building an artifact with the `dist-` builders? Which in turn requires longer build times and more builders?